### ISSUE TITLE
Render Single Block is not working

### ISSUE THREAD

**Issue by atulmy** [BomdiZane/EditorJS-React-Renderer#23]
When using example mentioned https://github.com/BomdiZane/EditorJS-React-Renderer#render-single-block I'm getting following error:
`TypeError: Cannot read property 'disableDefaultStyle' of undefined`

**Comment by atulmy** [comment]
Was able to temporarily fix by passing `config={{ disableDefaultStyle: false }}`:

```jsx
<ListOutput data={ listData } style={ style } config={{ disableDefaultStyle: false }} />
```

**Comment by dev-juju** [comment]
@atulmy What version are you suing?

**Comment by atulmy** [comment]
```
...
"@editorjs/editorjs": "^2.19.1",
"@editorjs/embed": "^2.4.0",
"@editorjs/header": "^2.6.1",
"@editorjs/image": "^2.6.0",
"@editorjs/list": "^1.6.1",
"@editorjs/paragraph": "^2.8.0",
"@editorjs/quote": "^2.4.0",
"@editorjs/underline": "^1.0.0",
...
"editorjs-react-renderer": "^2.5.0",
...
```

**Comment by dev-juju** [comment]
@atulmy 

Thanks for raising this.
Resolved in v2.7.2
